contrib/slurm-test/slurm_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+docker-compose up -d
+docker ps
+docker cp toil_workflow.py slurm-test_slurmmaster_1:/home/admin
+ln -s /toil/src/test/sort/sort.py sort.py
```

Comment:
Assuming we run this script here in `<repo root>/contrib/slurm-test`, then this file is 2 levels up to the main Toil directory, and from there in `src/test/sort/sort.py`. It's not going to be at an absolute path under `/toil`, generally.

```suggestion
ln -s ../../src/test/sort/sort.py sort.py
```

Really, though, I would commit the symlink to Git. Run that command from here and then `git add sort.py` and you won't have to worry about creating and destroying symlinks at test runtime.